From 7f9619a27980eed972820c37d8376fb622f5394e Mon Sep 17 00:00:00 2001
From: Brian Sheedy <bsheedy@chromium.org>
Date: Thu, 14 Jul 2022 23:11:16 +0000
Subject: [PATCH 089/110] [M102] Allow GPU M1 Macs to use Mac 12

The GPU M1 experimental builder seems happy with Mac 12, so allow the
stable configuration to use it as well in order to unblock the OS
upgrade.

(cherry picked from commit e718b25f38cec40abe16d6794c1a22c607f929bc)

Bug: 1335471
Change-Id: Ib82d5a45f9310de0669bebb57a2cf7e43072e353
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3760899
Auto-Submit: Brian Sheedy <bsheedy@chromium.org>
Reviewed-by: Yuly Novikov <ynovikov@chromium.org>
Commit-Queue: Brian Sheedy <bsheedy@chromium.org>
Commit-Queue: Yuly Novikov <ynovikov@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1023831}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3764173
Cr-Commit-Position: refs/branch-heads/5005@{#1248}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 testing/buildbot/chromium.gpu.fyi.json | 60 +++++++++++++-------------
 testing/buildbot/mixins.pyl            |  2 +-
 2 files changed, 31 insertions(+), 31 deletions(-)

diff --git a/testing/buildbot/chromium.gpu.fyi.json b/testing/buildbot/chromium.gpu.fyi.json
index 3c22537f9264..5de13f8a85dc 100644
--- a/testing/buildbot/chromium.gpu.fyi.json
+++ b/testing/buildbot/chromium.gpu.fyi.json
@@ -12871,7 +12871,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -12900,7 +12900,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -12927,7 +12927,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -12954,7 +12954,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -12978,7 +12978,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13002,7 +13002,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13023,7 +13023,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13048,7 +13048,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13086,7 +13086,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13122,7 +13122,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13158,7 +13158,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13194,7 +13194,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13234,7 +13234,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13279,7 +13279,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13324,7 +13324,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13360,7 +13360,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13396,7 +13396,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13441,7 +13441,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13486,7 +13486,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13513,7 +13513,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13549,7 +13549,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13586,7 +13586,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13622,7 +13622,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13660,7 +13660,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13699,7 +13699,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13738,7 +13738,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13776,7 +13776,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13814,7 +13814,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13852,7 +13852,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
@@ -13889,7 +13889,7 @@
             {
               "cpu": "arm64",
               "display_attached": "1",
-              "os": "Mac-11",
+              "os": "Mac-11|Mac-12",
               "pool": "chromium.tests"
             }
           ],
diff --git a/testing/buildbot/mixins.pyl b/testing/buildbot/mixins.pyl
index e628d94166d1..4feb8b7b3a6d 100644
--- a/testing/buildbot/mixins.pyl
+++ b/testing/buildbot/mixins.pyl
@@ -813,7 +813,7 @@
     'swarming': {
       'dimensions': {
         'cpu': 'arm64',
-        'os': 'Mac-11',
+        'os': 'Mac-11|Mac-12',
         'pool': 'chromium.tests',
         'display_attached': '1',
       },
-- 
2.38.1

